pages/index.js

```diff
@@ -2,7 +2,7 @@ import Head from 'next/head'
 import Header from '@components/Header'
 import Footer from '@components/Footer'
 
-export default function Home() {
+export default function Home({locale}) {
```

Comment:
You do not need to actually pass the locale, we use it in the demo for visual purposes.